src/theme/mifosx-theme.scss

```
@@ -68,7 +68,20 @@ $mifosx-app-theme: mat.m2-define-light-theme(
 @include shares-account-view-component-theme($mifosx-app-theme);
 @include account-view-component-theme($mifosx-app-theme);
 @include view-account-transfer-component-theme($mifosx-app-theme);
-@include copilot-component-theme($mifosx-app-theme);
+// Scoped, where the other component themes above are not, and deliberately.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)